### PR TITLE
Renaming the GitHub repository to `screwdriver`

### LINKED ISSUE #27: Rename to screwdriver?
I noticed that both kubernetes and spinnaker have their main repository be named after the git organization/product.  Should we do the same?

If so, the API would be the primary component of SD and as such we should rename it.

### PR DESCRIPTION
**Pull request by stjohnjohnson**
Resolves #27

### CODE REVIEW

**Comment by FenrirUnbound** [comment]
👍 

Looks good to me.

As an aside, our "create a new repository" documentation states that we should do a manual `npm publish` step. Do we need to also perform a manual publish, or will the pipeline do it automagically?

**Comment by d2lam** [comment]
👍  I assume you will change the repo name after this PR is merged?

**Comment by stjohnjohnson** [comment]
@FenrirUnbound I'm not renaming the module, it's still called `screwdriver-api`, but the repository will be renamed.  We could rename the module `screwdriverd` :P

File: README.md [hunk 1]
```diff
@@ -117,8 +117,8 @@ Code licensed under the BSD 3-Clause license. See LICENSE file for terms.
 [docker-stars]: https://img.shields.io/docker/stars/screwdrivercd/api.svg
 [docker-url]: https://hub.docker.com/r/screwdrivercd/api/
 [license-image]: https://img.shields.io/npm/l/screwdriver-api.svg
-[issues-image]: https://img.shields.io/github/issues/screwdriver-cd/api.svg
-[issues-url]: https://github.com/screwdriver-cd/api/issues
+[issues-image]: https://img.shields.io/github/issues/screwdriver-cd/screwdriver.svg
+[issues-url]: https://github.com/screwdriver-cd/screwdriver/issues
 [wercker-image]: https://app.wercker.com/status/3b34e93cc47c1b05d484158c012cb731
 [wercker-url]: https://app.wercker.com/project/bykey/3b34e93cc47c1b05d484158c012cb731
 [daviddm-image]: https://david-dm.org/screwdriver-cd/api.svg?theme=shields.io
```
Comment (okuryu) [inline README.md]: You might have forgotten to change here.

Comment (stjohnjohnson) [inline README.md]: Good catch, I thought daviddm was based on npm package.

**Comment by Filbird** [comment]
👍  Looks good!

**Pull request merged**